docs/index.md

```diff
@@ -74,3 +74,7 @@ with server.auth.sign_in(tableau_auth):
     print("\nThere are {} datasources on site: ".format(pagination_item.total_available))
     print([datasource.name for datasource in all_datasources])
 ```
+For Tableau online TSC.TableauAuth should be used with site_id
+```py
+tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD','SITEID')
+```
```

Comment:
Even though the parameter is called "site_id" in the TSC library, in the example we should use 'SITENAME' as the placeholder label because it maps to the REST API site name element and NOT the REST API site id element.

Reply:
Its even more confusing that that...

From the "QuerySites" API the value you want is in the "content-url" field.
In the Tableau Server Web Application it is called "Site ID".  

Dan makes a good point thought.  We should call it content-url and probably fix the naming in the UX.

Reply:
Yeah its the site name.

Reply:
More changes added .Please review them.

Reply:
Somehow a previous comment I made got lost. I want to strongly suggest that we make things simpler by treating the default site of a local Server installation as the exception, rather than describing all the other cases and giving a second sub-example. Below is my proposal, to which I added a sentence to address the confusion Irwin describes:

```py
import tableauserverclient as TSC
tableau_auth = TSC.TableauAuth('USERNAME', 'PASSWORD', 'SITENAME')
server = TSC.Server('http://SERVER_URL')
with server.auth.sign_in(tableau_auth):
    all_datasources, pagination_item = server.datasources.get()
    print("\nThere are {} datasources on site: ".format(pagination_item.total_available))
    print([datasource.name for datasource in all_datasources])
```
```SITENAME``` is same as the content URL of your site (though the parameter is called site_id in some Tableau tooling). This parameter can be omitted for the Default site of a locally installed Tableau Server.

Reply:
"Locally installed" is slightly off, because it could be 'remote' to the script but it's still not Tableau Online. I would call it "On Premise" which is how we generally refer to self-installed Server.